modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -198,7 +198,44 @@ controller-0$ sudo docker restart octavia_worker
 
 [NOTE]
 ====
-Depending on your OpenStack environment, Octavia might not support UDP
-listeners, which means there is no support for UDP Services if Kuryr SDN is
-used.
+Depending on your OpenStack environment, Octavia may not support UDP
+listeners. If you use Kuryr SDN on OpenStack version 15 or earlier, UDP services are not supported. 
+{rh-openstack} version 16 or later support UDP.
 ====
+
+
+== The Octavia OVN Driver
+
+Octavia supports multiple provider drivers through the Octavia API.
+
+To see all available Octavia provider drivers, on a command line, enter:
+----
+$ openstack loadbalancer provider list
+----
+
+The result is a list of drivers:
+----
++---------+-------------------------------------------------+
+| name    | description                                     |
++---------+-------------------------------------------------+
+| amphora | The Octavia Amphora driver.                     |
+| octavia | Deprecated alias of the Octavia Amphora driver. |
+| ovn     | Octavia OVN driver.                             |
++---------+-------------------------------------------------+
+----
+
+Beginning with {rh-openstack} version 16, the Octavia OVN provider driver (`ovn`) is supported on
+{product-title} on {rh-openstack} deployments.
+
+`ovn` is an integration driver for the load balancing
+that Octavia and OVN provide. It supports basic load balancing capabilities,
+and is based on OpenFlow rules. The driver is automatically enabled
+in Octavia by Director on deployments that use OVN Neutron ML2.
+
+The Amphora provider driver is the default driver. However, if `ovn` is enabled, Kuryr uses it.
```

Comment:
s/However, if `ovn` is enabled, Kuryr uses it./If `ovn` is enabled, however, Kuryr uses it.